test-dts/defineComponent.test-d.tsx

```diff
@@ -717,6 +718,8 @@ describe('extends with mixins', () => {
   // Test TSX
   expectType<JSX.Element>(<MyComponent mP1="p1" mP2 mP3 p1 p2={1} p3 z={'z'} />)
 
+  expectType<JSX.Element>(<MyComponent mP1="p1" mP2 mP3 p1 p2={1} p3 z={'z'} />)
```

Comment:
Does this duplicate the line above?

Reply:
Was debugging something and didn't clean up 😅 
Good catch :)